source/Jobbr.Server.UnitTests/Storage/InMemoryJobStorageProviderTests.cs

```diff
@@ -0,0 +1,412 @@
+
+            // Assert
+            result.ShouldNotBeNull().ShouldNotBe(trigger);
+            result.ShouldBeOfType<RecurringTrigger>();
+            result.JobId.ShouldBe(jobId);
+        }
+
+        [TestMethod]
+        public void GetJobRunById_ShouldReturnClone()
+        {
+            // Arrange
+            var jobRun = new JobRun();
+            _provider.AddJobRun(jobRun);
+
+            // Act
+            var result = _provider.GetJobRunById(jobRun.Id);
+
+            // Assert
+            result.ShouldNotBeNull().ShouldNotBe(jobRun);
+        }
+
+        [TestMethod]
+        public void DisableTrigger_ShouldSetIsActive()
+        {
+            // Arrange
+            var jobId = 1;
+            var trigger = new RecurringTrigger { IsActive = true };
+            _provider.AddTrigger(jobId, trigger);
+
+            // Act
+            _provider.DisableTrigger(jobId, trigger.Id);
+
+            // Assert
+            trigger.IsActive.ShouldBeFalse();
+        }
+
+        [TestMethod]
+        public void DisableTrigger_BadId_ShouldThrow()
+        {
+            // Arrange
+            var jobId = 1;
+            var trigger = new RecurringTrigger { IsActive = true };
+            _provider.AddTrigger(jobId, trigger);
+
+            // Act & Assert
+            Should.Throw<InvalidOperationException>(() => _provider.DisableTrigger(jobId, trigger.Id + 1));
+        }
+
+        [TestMethod]
+        public void EnableTrigger_ShouldSetIsActive()
+        {
+            // Arrange
+            var jobId = 1;
+            var trigger = new RecurringTrigger { IsActive = false };
+            _provider.AddTrigger(jobId, trigger);
+
+            // Act
+            _provider.EnableTrigger(jobId, trigger.Id);
+
+            // Assert
+            trigger.IsActive.ShouldBeTrue();
+        }
+
+        [TestMethod]
+        public void EnableTrigger_BadId_ShouldThrow()
+        {
+            // Arrange
+            var jobId = 1;
+            var trigger = new RecurringTrigger { IsActive = false };
+            _provider.AddTrigger(jobId, trigger);
+
+            // Act & Assert
+            Should.Throw<InvalidOperationException>(() => _provider.EnableTrigger(jobId, trigger.Id + 1));
+        }
+
+        [TestMethod]
+        public void Update_Job_ShouldReplaceExisting()
+        {
+            // Arrange
+            var job = new Job();
+            _provider.AddJob(job);
+
+            var jobUpdate = new Job { Id = job.Id };
+            var identifier = "my-unique-identifier";
+            jobUpdate.UniqueName = identifier;
+            var jobCount = _provider.GetJobsCount();
+
+            // Act
+            _provider.Update(jobUpdate);
+
+            // Assert
+            _provider.GetJobsCount().ShouldBe(jobCount);
+            job = _provider.GetJobById(job.Id);
+            job.UniqueName.ShouldBe(jobUpdate.UniqueName);
+        }
+
+        [Ignore]
```

Comment:
Why ignore?

Reply:
Can you add a justification to the attribute, so it's clearer: `[Ignore("blabla")]`